radio/src/pulses/pxx.h

```diff
@@ -29,7 +29,7 @@
 
 #define PXX2_LOWSPEED_BAUDRATE             230400
 #define PXX2_HIGHSPEED_BAUDRATE            450000
-#define PXX2_PERIOD                        4 // 4ms
+#define PXX2_PERIOD                        50 // 4ms
```

Comment:
why 50?

Reply:
You suggested me change it to 50ms . Racing mode is 4.6ms , we should set it more then 4.6ms

Reply:
50 doesn't look right at all to me

Reply:
right, I have pushed a proposal, it needs testing (internal module + heartbeat)